evap/evaluation/models.py

```diff
@@ -1599,6 +1599,15 @@ def send_contributor_publish_notifications(cls, evaluations, template=None):
 
         evaluations_per_contributor = defaultdict(set)
         for evaluation in evaluations:
+            # an average grade is published or a general text answer exists
+            relevant_information_published_for_responsibles = (
```

Comment:
it seems like you're only using this once. why didn't you move it into the if?

Reply:
I moved it out to prevent having a multi-line condition in the if (which I always struggle to format nicely) -- with the side benefit, that the condition now has a semantic name.